File: zhiyoufy-server-java/zhiyoufy-server/src/main/java/com/example/zhiyoufy/server/domain/dto/pms/PmsJobTemplateUpdateParam.java
package com.example.zhiyoufy.server.domain.dto.pms;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
@JsonInclude(JsonInclude.Include.NON_NULL)
public class PmsJobTemplateUpdateParam {
	private String name;

	private String description;

	private String jobPath;

	private String workerLabels;

	private Integer timeoutSeconds;

	private String baseConfPath;

	private String privateConfPath;

	private String configSingles;
	private String configCollections;

	private String extraArgs;

	private Boolean isPerf;
	private String dashboardAddr;
}

File: zhiyoufy-web/src/model/dto/pms/PmsJobTemplateUpdateParam.ts
export interface PmsJobTemplateUpdateParam {
  name?: string;
  description?: string;
  jobPath?: string;
  workerLabels?: string;
  timeoutSeconds?: number;
  baseConfPath?: string;
  privateConfPath?: string;
  configSingles?: string;
  configCollections?: string;
  extraArgs?: string;
  isPerf?: boolean;
  dashboardAddr?: string;
}
